react/features/base/conference/constants.js

```diff
@@ -1,16 +1,16 @@
 /**
- * The audio only state where audio only mode is enabled and cannot be disabled.
+ * The audio only state where audio only mode is enabled and can be disabled.
  *
  * @type {string}
  */
-export const AUDIO_ONLY_ENABLED_PERMANENTLY = 'audio-only-enabled-with-lock';
+export const AUDIO_ONLY_ENABLED = 'audio-only-enabled-without-lock';
```

Comment:
should we change the name of the field to match the new variable names? ('permanent' vs 'lock')